backend/metrics/calculations.py

```diff
@@ -46,7 +46,7 @@ def calculate_hospitalizations(
     return df
 
 
-def _moving_average_grouped(df: pd.DataFrame, group_columns: list[str], target_column: str, window_size) -> pd.Series:
```

Comment:
please keep this. upgrade to python 3.8 or 3.9...
Or if you ahve to switch to captital L List and import it from typing
from typing import List

Reply:
Added back